docs/api.md

```diff
@@ -60,6 +60,10 @@ empty
 | 202   | |
 | 500 | something went wrong (see logs of the simulator server) |
 
+---
+## **deprecated APIs**
+
+The following APIs are deprecated. They will be removed in the near future.
 ## Apply Node
```

Comment:
Could you change all deprecated APIs titles like this?

```suggestion
### Apply Node
```

Reply:
fixed: [4d227ee](https://github.com/kubernetes-sigs/kube-scheduler-simulator/pull/127/commits/4d227ee0e035741bd10e0ad809bd2ee610ced078)